src/app/container/container.component.html

```diff
@@ -151,8 +151,14 @@ <h3>
       </tab>
       <tab heading="Files" (select)="setEntryTab('files')">
         <div *ngIf="validVersions">
-          <app-files-container [entrypath]="tool?.tool_path" [id]="tool?.id" [versions]="validVersions" [default]="defaultVersion" [selectedVersion]="selectedVersion">
-          </app-files-container>
+          <div *ngIf="tool?.mode !== DockstoreToolType.ModeEnum.HOSTED">
```

Comment:
That's an interesting thought. 
Without a versions tab, where do we see verification info for hosted workflows/tools?